server/openejb-cxf/pom.xml

```diff
@@ -264,7 +264,7 @@
     <dependency>
       <groupId>com.sun.xml.messaging.saaj</groupId>
       <artifactId>saaj-impl</artifactId>
-      <version>${saaj-impl.version}</version>
+      <version>${version.impl.saaj}</version>
```

Comment:
`version.saaj.impl` or `version.saaj-impl` ?

Reply:
The intent was to have all future ambiguous impl in a "version.impl" prefix like version.impl.activation and version.impl.mail to disambiguate with api version with prefixes without impl. There should be ambiguity only with those three. It can be changed to something else entirely if needed.